### PR TITLE
tidb-test: add prisma integration test

### PR DESCRIPTION
**Pull request by Mini256**
- Add Prisma (Node.js ORM framework) Integration Test to tidb-test Pipeline

### CODE REVIEW

**Comment by Mini256** [comment]
/hold

Wait PR https://github.com/pingcap/tidb-test/pull/2163 has been merged

**Review by purelind — APPROVED** [review]
/lgtm

File: pipelines/pingcap/tidb-test/latest/ghpr_integration_prisma_test.groovy [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                    }
+                }
+            }
+        }
+        stage('Prisma Tests') {
+            matrix {
+                axes {
+                    axis {
+                        name 'TEST_PARAMS'
+                        values 'prisma_test ./test.sh',
```
Comment (purelind) [inline pipelines/pingcap/tidb-test/latest/ghpr_integration_prisma_test.groovy:91]: There is an extra comma.

**Review by purelind — APPROVED** [review]

**Pull request merged**